---
result: FAILURE
url: http://wilson.ci.chef.co/job/chef-server-12-trigger-ad_hoc/13/
failure_category: code
failure_cause: 'chef-client: ubuntu-10.04, ubuntu-12.04: chef-server-12-test'
timestamp: 2015-12-29 16:39:06 UTC
duration: 2h3m32s
triggered_by: jaym
active_duration: 2h3m16s
queue_delays: 10s
parameters:
  GIT_REF: jdm/fips-final
  EXPIRE_CACHE: false
change:
  git_remote: git@github.com:chef/chef-server.git
  git_commit: dd6db4a69367af7c80ef4beb2e6e7fffc4de163a
  project: chef-server
  version: 12.3.1+20151229163921
stages:
  chef-server-12-test:
    result: FAILURE
    failure_category: code
    failure_cause: 'chef-client: ubuntu-10.04, ubuntu-12.04'
    url: http://wilson.ci.chef.co/job/chef-server-12-test/22/
    duration: 50m46s
    queue_delay: 10s
    runs:
      ubuntu-10.04:
        result: FAILURE
        failure_category: code
        failure_cause: chef-client
        failed_in:
          step: chef-client
        url: http://wilson.ci.chef.co/job/chef-server-12-test/architecture=x86_64,platform=ubuntu-10.04,project=chef-server,role=tester/22/
        duration: 5m9s
        logs:
          consoleText:
            967: |
              [2015-12-29T17:54:55+00:00] INFO: Processing execute[restart_rabbitmq_log_service] action run (private-chef::rabbitmq line 19)
              [2015-12-29T17:54:56+00:00] INFO: execute[restart_rabbitmq_log_service] ran successfully
              [2015-12-29T17:54:56+00:00] ERROR: Running exception handlers
              [2015-12-29T17:54:56+00:00] ERROR: Exception handlers complete
              [2015-12-29T17:54:56+00:00] FATAL: Stacktrace dumped to /opt/opscode/embedded/cookbooks/cache/chef-stacktrace.out
              [2015-12-29T17:54:56+00:00] FATAL: Please provide the contents of the stacktrace.out file if you file a bug report
              [2015-12-29T17:54:56+00:00] FATAL: Mixlib::ShellOut::ShellCommandFailed: execute[/opt/opscode/embedded/bin/chpst -u opscode -U opscode /opt/opscode/embedded/bin/rabbitmqctl wait /var/opt/opscode/rabbitmq/db/rabbit@localhost.pid] (private-chef::rabbitmq line 110) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected process to exit with [0], but received '2'
              ---- Begin output of /opt/opscode/embedded/bin/chpst -u opscode -U opscode /opt/opscode/embedded/bin/rabbitmqctl wait /var/opt/opscode/rabbitmq/db/rabbit@localhost.pid ----
              STDOUT: Waiting for rabbit@localhost ...
            990: |
              No entry for terminal type "unknown";
              using dumb terminal settings.
              /opt/opscode/embedded/service/oc-chef-pedant/lib/pedant/config.rb:36:in `from_argv': Configuration file '/var/opt/opscode/oc-chef-pedant/etc/pedant_config.rb' not found! (RuntimeError)
                from /opt/opscode/embedded/service/oc-chef-pedant/lib/pedant.rb:54:in `setup'
                from ./bin/oc-chef-pedant:10:in `<main>'
              Build step 'Invoke XShell command' marked build as failure
              Performing Post build task...
              Match found for : : True
            1014: |
              Recording fingerprints
              Recording test results
              ERROR: Publisher hudson.tasks.junit.JUnitResultArchiver aborted due to exception
              hudson.AbortException: No test report files were found. Configuration error?
                at hudson.tasks.junit.JUnitParser$ParseResultCallable.invoke(JUnitParser.java:116)
                at hudson.tasks.junit.JUnitParser$ParseResultCallable.invoke(JUnitParser.java:92)
                at hudson.FilePath$FileCallableWrapper.call(FilePath.java:2688)
                at hudson.remoting.UserRequest.perform(UserRequest.java:121)
                at hudson.remoting.UserRequest.perform(UserRequest.java:49)
                at hudson.remoting.Request$2.run(Request.java:325)
                at hudson.remoting.InterceptingExecutorService$1.call(InterceptingExecutorService.java:68)
                at java.util.concurrent.FutureTask.run(FutureTask.java:262)
                at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
                at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
                at java.lang.Thread.run(Thread.java:745)
                at ......remote call to chef-server-ubuntu-10.04-tester-f80d1f(Native Method)
                at hudson.remoting.Channel.attachCallSiteStackTrace(Channel.java:1360)
                at hudson.remoting.UserResponse.retrieve(UserRequest.java:221)
                at hudson.remoting.Channel.call(Channel.java:753)
                at hudson.FilePath.act(FilePath.java:978)
                at hudson.FilePath.act(FilePath.java:967)
                at hudson.tasks.junit.JUnitParser.parseResult(JUnitParser.java:89)
                at hudson.tasks.junit.JUnitResultArchiver.parse(JUnitResultArchiver.java:121)
                at hudson.tasks.junit.JUnitResultArchiver.perform(JUnitResultArchiver.java:138)
                at hudson.tasks.BuildStepCompatibilityLayer.perform(BuildStepCompatibilityLayer.java:74)
                at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
                at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:761)
                at hudson.model.AbstractBuild$AbstractBuildExecution.performAllBuildSteps(AbstractBuild.java:721)
                at hudson.model.Build$BuildExecution.post2(Build.java:183)
                at hudson.model.AbstractBuild$AbstractBuildExecution.post(AbstractBuild.java:670)
                at hudson.model.Run.execute(Run.java:1766)
                at hudson.matrix.MatrixRun.run(MatrixRun.java:146)
                at hudson.model.ResourceController.execute(ResourceController.java:98)
                at hudson.model.Executor.run(Executor.java:374)
              [BFA] Scanning build for known causes...
              [BFA] No failure causes found
        steps:
          total: 5m9s
          before (setup time): 2m10s
          after (cleanup time): 2m54s
      ubuntu-12.04:
        result: FAILURE
        failure_category: code
        failure_cause: chef-client
        failed_in:
          step: chef-client
        url: http://wilson.ci.chef.co/job/chef-server-12-test/architecture=x86_64,platform=ubuntu-12.04,project=chef-server,role=tester/22/
        duration: 7m15s
        logs:
          consoleText:
            966: |
              [2015-12-29T17:56:47+00:00] INFO: Processing execute[restart_rabbitmq_log_service] action run (private-chef::rabbitmq line 19)
              [2015-12-29T17:56:47+00:00] INFO: execute[restart_rabbitmq_log_service] ran successfully
              [2015-12-29T17:56:47+00:00] ERROR: Running exception handlers
              [2015-12-29T17:56:47+00:00] ERROR: Exception handlers complete
              [2015-12-29T17:56:47+00:00] FATAL: Stacktrace dumped to /opt/opscode/embedded/cookbooks/cache/chef-stacktrace.out
              [2015-12-29T17:56:47+00:00] FATAL: Please provide the contents of the stacktrace.out file if you file a bug report
              [2015-12-29T17:56:47+00:00] FATAL: Mixlib::ShellOut::ShellCommandFailed: execute[/opt/opscode/embedded/bin/chpst -u opscode -U opscode /opt/opscode/embedded/bin/rabbitmqctl wait /var/opt/opscode/rabbitmq/db/rabbit@localhost.pid] (private-chef::rabbitmq line 110) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected process to exit with [0], but received '2'
              ---- Begin output of /opt/opscode/embedded/bin/chpst -u opscode -U opscode /opt/opscode/embedded/bin/rabbitmqctl wait /var/opt/opscode/rabbitmq/db/rabbit@localhost.pid ----
              STDOUT: Waiting for rabbit@localhost ...
            989: |
              No entry for terminal type "unknown";
              using dumb terminal settings.
              /opt/opscode/embedded/service/oc-chef-pedant/lib/pedant/config.rb:36:in `from_argv': Configuration file '/var/opt/opscode/oc-chef-pedant/etc/pedant_config.rb' not found! (RuntimeError)
                from /opt/opscode/embedded/service/oc-chef-pedant/lib/pedant.rb:54:in `setup'
                from ./bin/oc-chef-pedant:10:in `<main>'
              Build step 'Invoke XShell command' marked build as failure
              Performing Post build task...
              Match found for : : True
            1013: |
              Recording fingerprints
              Recording test results
              ERROR: Publisher hudson.tasks.junit.JUnitResultArchiver aborted due to exception
              hudson.AbortException: No test report files were found. Configuration error?
                at hudson.tasks.junit.JUnitParser$ParseResultCallable.invoke(JUnitParser.java:116)
                at hudson.tasks.junit.JUnitParser$ParseResultCallable.invoke(JUnitParser.java:92)
                at hudson.FilePath$FileCallableWrapper.call(FilePath.java:2688)
                at hudson.remoting.UserRequest.perform(UserRequest.java:121)
                at hudson.remoting.UserRequest.perform(UserRequest.java:49)
                at hudson.remoting.Request$2.run(Request.java:325)
                at hudson.remoting.InterceptingExecutorService$1.call(InterceptingExecutorService.java:68)
                at java.util.concurrent.FutureTask.run(FutureTask.java:262)
                at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
                at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
                at java.lang.Thread.run(Thread.java:745)
                at ......remote call to chef-server-ubuntu-12.04-tester-5b0c0e(Native Method)
                at hudson.remoting.Channel.attachCallSiteStackTrace(Channel.java:1360)
                at hudson.remoting.UserResponse.retrieve(UserRequest.java:221)
                at hudson.remoting.Channel.call(Channel.java:753)
                at hudson.FilePath.act(FilePath.java:978)
                at hudson.FilePath.act(FilePath.java:967)
                at hudson.tasks.junit.JUnitParser.parseResult(JUnitParser.java:89)
                at hudson.tasks.junit.JUnitResultArchiver.parse(JUnitResultArchiver.java:121)
                at hudson.tasks.junit.JUnitResultArchiver.perform(JUnitResultArchiver.java:138)
                at hudson.tasks.BuildStepCompatibilityLayer.perform(BuildStepCompatibilityLayer.java:74)
                at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
                at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:761)
                at hudson.model.AbstractBuild$AbstractBuildExecution.performAllBuildSteps(AbstractBuild.java:721)
                at hudson.model.Build$BuildExecution.post2(Build.java:183)
                at hudson.model.AbstractBuild$AbstractBuildExecution.post(AbstractBuild.java:670)
                at hudson.model.Run.execute(Run.java:1766)
                at hudson.matrix.MatrixRun.run(MatrixRun.java:146)
                at hudson.model.ResourceController.execute(ResourceController.java:98)
                at hudson.model.Executor.run(Executor.java:374)
              [BFA] Scanning build for known causes...
              [BFA] No failure causes found
        steps:
          total: 7m15s
          before (setup time): 3m25s
          after (cleanup time): 3m45s
      el-5:
        result: SUCCESS
        url: http://wilson.ci.chef.co/job/chef-server-12-test/architecture=x86_64,platform=el-5,project=chef-server,role=tester/22/
        duration: 50m45s
        steps:
          total: 50m45s
          before (setup time): 4m36s
          chef-client private-chef::default: 7m22s
          after (cleanup time): 36m41s
      el-6:
        result: SUCCESS
        url: http://wilson.ci.chef.co/job/chef-server-12-test/architecture=x86_64,platform=el-6,project=chef-server,role=tester/22/
        duration: 47m22s
        steps:
          total: 47m22s
          before (setup time): 2m1s
          chef-client private-chef::default: 3m34s
          after (cleanup time): 39m38s
      el-7:
        result: SUCCESS
        url: http://wilson.ci.chef.co/job/chef-server-12-test/architecture=x86_64,platform=el-7,project=chef-server,role=tester/22/
        duration: 40m10s
        steps:
          total: 40m10s
          before (setup time): 3m9s
          chef-client private-chef::default: 4m46s
          after (cleanup time): 30m8s
      ubuntu-14.04:
        result: SUCCESS
        url: http://wilson.ci.chef.co/job/chef-server-12-test/architecture=x86_64,platform=ubuntu-14.04,project=chef-server,role=tester/22/
        duration: 41m33s
        steps:
          total: 41m33s
          before (setup time): 2m20s
          chef-client private-chef::default: 2m32s
          after (cleanup time): 34m34s
  chef-server-12-build:
    result: SUCCESS
    url: http://wilson.ci.chef.co/job/chef-server-12-build/23/
    duration: 1h12m22s
    runs:
      el-5:
        result: SUCCESS
        url: http://wilson.ci.chef.co/job/chef-server-12-build/architecture=x86_64,platform=el-5,project=chef-server,role=builder/23/
        duration: 1h10m42s
      el-6:
        result: SUCCESS
        url: http://wilson.ci.chef.co/job/chef-server-12-build/architecture=x86_64,platform=el-6,project=chef-server,role=builder/23/
        duration: 1h11m22s
      el-7:
        result: SUCCESS
        url: http://wilson.ci.chef.co/job/chef-server-12-build/architecture=x86_64,platform=el-7,project=chef-server,role=builder/23/
        duration: 59m24s
      ubuntu-10.04:
        result: SUCCESS
        url: http://wilson.ci.chef.co/job/chef-server-12-build/architecture=x86_64,platform=ubuntu-10.04,project=chef-server,role=builder/23/
        duration: 46m48s
  chef-server-12-trigger-ad_hoc:
    result: SUCCESS
    url: http://wilson.ci.chef.co/job/chef-server-12-trigger-ad_hoc/13/
    duration: 7s
